src/domain/accounts/services/Cargo.toml

```diff
@@ -23,12 +23,15 @@ doctest = false
 
 [dependencies]
 database-common = { workspace = true }
+email-gateway = { workspace = true }
 init-on-startup = { workspace = true }
 internal-error = { workspace = true }
 kamu-accounts = { workspace = true }
+messaging-outbox = { workspace = true }
 odf = { workspace = true }
 time-source = { workspace = true }
 
+askama = { version = "0.12" }
```

Comment:
Hm, I expected to see this dependency in an infra layer, not in a domain crate

Reply:
I don't agree, Askama is not something of infrastructure layer. It's basically a string conversion by temlate.
No matter what is the technology to send an email is, it is still an HTML page.

Reply:
But if we move the email generation to Node, then everything about `Askama` and emails, basically 80% of this branch, moves to Node.